documentation/docs/charges.mdx

```diff
@@ -3,40 +3,65 @@ id: charges
 title: Charges
 ---
 
-Charges represent objects with which Clients can instruct Wallets to perform a Payment to an Invoice within the scope
-of a Mandate. Wallets MUST implement the Charges API on conjunction with Mandates.
+Charges represent instructions by a client to perform a payment from the account
+against which the mandate was created. The existence of the mandate does not
+imply that the client has authorization to initiate payments, the client MUST
+have authorization to create charges against the mandate.
 
-| Property    | Type           |
-|-------------|----------------|
-| id          | UUID           |
-| invoice     | int64          |
-| mandate     | int64          |
-| status      | enum           |
+## Resource Representation
+
+The `application/json` representation of an invoice resource provides the
+details of the invoice as a JSON document.
+
+| Property | Type | Description                                        |
+| -------- | ---- | -------------------------------------------------- |
+| id       | URL  | The URL identifying the invoice                    |
+| mandate  | URL  | The mandate against which the charge is being made |
+| invoice  | URL  | The invoice which is being paid via this charge    |
+| status   | enum | TBD                                                |
 
 ## APIs
 
+The charges API endpoint is relative to the mandate that is being charged using
+the sub-resource URL: **`./charges`**
+
+Specifically, the URL of the charges API endpoint is the result of performing
+the URL parsing algorithm in
+[WHATWG URL](https://url.spec.whatwg.org/#url-parsing) where:
+
+- _input_ is `./charges`, and
+- _base_ is the mandate URL
+
+For example, for the mandate URL `https://wallet.example/mandates/7236rf76` the
+charges API endpoint for that mandate is
+`https://wallet.example/mandates/7236rf76/charges`
+
 ### Create
```

Comment:
This endpoint needs idempotency. I wonder if the invoice used is sufficient to define the idempotent behavior expected.

Reply:
To clarify then: there can be multiple charges per mandate, but each charge requires a unique invoice?